src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -37,6 +38,12 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
     }
 
     /**
+     * Use the new allele frequency / QUAL score model
+     */
+    @Argument(fullName = "useNewAFCalculator", shortName = "newQual", doc = "If provided, we will use the new AF model instead of the so-called exact model", optional = true)
+    public boolean USE_NEW_AF_CALCULATOR = false;
```

Comment:
With HaplotypeCaller one can choose the implementation using the ```pnrm``` argument which is defined in ```StandardCallerArgumentCollection```. Unfortunately this is not the case for ```GenotypeGVCFs```, this does not make much sense. Could you please add this argument to ```GenotypeGVCFs``` so that the user can use the same argument to select the implementation by name (old or new)? 

Perhaps you want to get ```StandardCallerArgumentCollection``` to delegate the argument definition to ```GenotypeCalculationArgumentCollection``` as is likely that you bump into trouble getting ```GenotypeGVCFs``` to absorve all the arguments in ```StandardCallerArgumentCollection```.